.github/workflows/build.yml

```
@@ -55,13 +55,22 @@ jobs:
           - splitreus62_right
           - tg4x
           - tidbit
+        build-arg: [""]
```

```suggestion
        build-args: [""]
```
I can see this argument turning into a catch-all argument, so perhaps we should make it plural?

Also, I guess it's technically `cmake-args` (perhaps?) as it's after the `--`?  That said, I'm not sure what terminology is used to describe the group of arguments before the `--`!?

---

`cmake-args` also seems to match well with: https://docs.zephyrproject.org/2.4.0/guides/west/build-flash-debug.html#permanent-cmake-arguments

---

I think `cmake-args` works for me. Those other things would be `west-args`.

---

@innovaker Updated to `cmake-args`